madgiraffestudios/src/components/Booking/Booking.js

```diff
@@ -80,39 +81,42 @@ const [message, setMessage] = useState ('');
   setMessage('');
  }
 
+
+
       return (
         <Modal isOpen={displayModal} toggle={() =>setDisplayModal(false)}   className="mt-5 text-center">
         <ModalHeader className="bg-primary text-white">Let's Work Together!</ModalHeader>
          <ModalBody className="font-weight-bold">
          <Form onSubmit={handleSubmit} >
-                 <FormGroup>
-                     <Label for="receipient name" className="d-block text-left">Full Name</Label>
-                     <Input type="text" name="fullName" value={fullName} onChange={e=>setFullName(e.target.value)} placeholder="Your name" required/>
+                 <FormGroup className="form-group required">
+                     <Label for="receipient name" className="d-block text-left control-label">Full Name</Label>
+                     <Input type="text" name="fullName" value={fullName} onChange={e=>setFullName(e.target.value)} placeholder="Your name" title="Please put your name for us" required />
                      <FormFeedback>You will not be able to see this</FormFeedback>
                  </FormGroup>
-                 <FormGroup>
-                     <Label for="email" className="d-block text-left">Email</Label>
-                     <Input type="email" name="email" value={email} onChange={e=>setEmail(e.target.value)} placeholder="Email@gmail.com" />
+                 <FormGroup className="form-group required">
+                     <Label for="email" className="d-block text-left control-label">Email</Label>
+                     <Input type="email" name="email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" value={email} onChange={e=>setEmail(e.target.value)} placeholder="Email@gmail.com" title="Please put in a valid email address" required />
                  </FormGroup>
-                 <FormGroup>
+                 <FormGroup >
                      <Label for="phone" className="d-block text-left">Phone</Label>
-                     <Input type="tel" name="phone" value={phone} onChange={e=>setPhone(e.target.value)} placeholder="(917)-000-0000" />
+                     <Input type="tel" name="phone" pattern="\d{3}[\-]\d{3}[\-]\d{4}" value={phone} onChange={e=>setPhone(e.target.value)} placeholder="917-000-0000" title="Please put in phone number format 917-XXX-XXX"/>
```

Comment:
standard placeholder for phones is 123-456-7890, please consider updating is per user experience and eliminating confusion

Reply:
will address in next PR

Reply:
changed in next PR